helpers/assignRecipesToCalendarDays.js

```diff
@@ -1,20 +1,20 @@
-import updateUserInDb from "./updateUserInDb";
+import updateHouseholdInDb from "./updateHouseholdInDb";
 
 export default async function assignRecipesToCalendarDays(
   assignments,
-  user,
-  mutateUser
+  household,
```

Comment:
We could potentially use [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) here (and then we don't need the if check below)

```suggestion
  household = [],
```